config/consensus.go

```diff
@@ -381,6 +381,8 @@ type ConsensusParams struct {
 	// 5. checking that in the case of going online the VoteFirst is less or equal to the LastValid+1.
 	// 6. checking that in the case of going online the VoteFirst is less or equal to the next network round.
 	EnableKeyregCoherencyCheck bool
+
+	EnableExPagesOnAppUpdate bool
```

Comment:
Since "Ex" reads to me like "no longer" (Like Ex Husband, or Ex Googler), I'd avoid this abbreviation, and just use Extra.